airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -360,7 +360,7 @@
         "type": "string",
         "default": "",
         "description": "Your S3 endpoint url. Read more <a href=\"https://docs.aws.amazon.com/general/latest/gr/s3.html#:~:text=Service%20endpoints-,Amazon%20S3%20endpoints,-When%20you%20use\">here</a>",
-        "examples": ["http://localhost:9000"],
+        "examples": ["s3-accesspoint.us-east-2.amazonaws.com"],
```

Comment:
is it intentional to remove the `http://` protocol prefix? I didn't see a change to reinject that, do we not need it when passing the endpoint to the s3 client object?

Reply:
It was initially intentional since previously the understanding was that users could not host their own S3 instance but that appears to not be the case with further reading of minio S3 hosted instances. When you say reinject that, can you clarify? 

Based on my understanding this was merely a change in the pre-filled example but not the default value which from what it seemed was `""`. Was my understanding incorrect and that this is actually where the user injects the `http` prefix string?